src/main/java/stringCalculator/StringCalculator.java

```diff
@@ -0,0 +1,75 @@
+package stringCalculator;
+
+import java.util.StringTokenizer;
+
+import static java.lang.Double.parseDouble;
+
+public class StringCalculator {
+
+    public double calculate(String text) {
+        if(text.equals(" ") || text.isEmpty()) throw new IllegalArgumentException("수식이 비었거나 null 입니다.");
+
+        StringTokenizer stringTokenizer = new StringTokenizer(text);
+
+        if(isPossibleToCalulateFirst(stringTokenizer)) throw new IllegalArgumentException("수식이 잘못되었습니다.");
```

Comment:
```suggestion
        if (isPossibleToCalulateFirst(stringTokenizer)) {
                throw new IllegalArgumentException("수식이 잘못되었습니다.");
        }
```

컨벤션은 IntelliJ Auto Formatting 기능을 사용하는 습관을 들이면 많은 도움이 됩니다!

The auto-formatting shortcut in IntelliJ

* For windows Ctrl + Alt + L.
* For ubuntu Ctrl + Alt + windows + L.
* For Mac Command + Option + L.